docs/variants/tuxedo_ibs15/custom_logo.md

```diff
@@ -0,0 +1,32 @@
+# Boot logo replacement instructions
```

Comment:
Why we breaking DRY rule? This site is duplication of https://docs.dasharo.com/variants/clevo_nv41/custom_logo/
When we programming do we copy function everywhere it is needed? Or we create library that can be linked as needed?

Reply:
Moved custom logo instructions to one common location for all platforms.